ledger-core/network/nds/msgdelivery/uniproto_utils_test.go

```diff
@@ -0,0 +1,109 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
+
+package msgdelivery
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/insolar/assured-ledger/ledger-core/network/nds/uniproto"
+	"github.com/insolar/assured-ledger/ledger-core/network/nds/uniproto/l2/uniserver"
+	"github.com/insolar/assured-ledger/ledger-core/network/nwapi"
+	"github.com/insolar/assured-ledger/ledger-core/pulse"
+	"github.com/insolar/assured-ledger/ledger-core/vanilla/cryptkit"
+	"github.com/insolar/assured-ledger/ledger-core/vanilla/longbits"
+)
+
+var (
+	servers = make([]*UnitProtoServer, 0)
+)
+
+func getServerByIndex(idx int) *UnitProtoServer {
+	if idx < 1 || idx > len(servers) {
+		panic("")
+	}
+
+	return servers[idx-1]
+}
+
+type UnitProtoServer struct {
```

Comment:
UniprotoServer - from word "unified" not from "unit"

Reply:
fixed